### DISCUSSION TITLE
Getting odd errors after dependencies update

### DISCUSSION THREAD

**Discussion by serp1412** [goatandsheep/react-native-dotenv#402] [Q&A]
Hey all,

We're seeing weird errors that appeared after dependencies change:

Error: ENOENT: no such file or directory, open '.env.development.local'
Error: ENOENT: no such file or directory, open '.env.development'
Error: ENOENT: no such file or directory, open '.env.local'

We didn't have this issue before the dependency update and we didn't change anything except the dependencies.

version in package.json: "react-native-dotenv": "^3.4.2"

it doesn't fail anything, just annoying to see this all of a sudden

anyone knows what this is about?

Cheers,

**Comment by goatandsheep** [comment]
Try `"verbose": false`?
